docs/gaia/private-testnet.md

```diff
@@ -1,4 +1,4 @@
-# Networks
+# Deploy a private testnet
```

Comment:
hm, they're not really "private", or intended to be. Maybe call them standalone testnets?